### ISSUE TITLE
pollreactor implements IReactorFDSet but doesn't z.i.implements() it

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11073]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)| @mwhudson reported|
|-|-|
|Trac ID|trac#1539|
|Type|enhancement|
|Created|2006-03-04 13:55:28Z|

It certainly looks to me like twisted.internet.pollreactor.PollReactor implements the twisted.internet.interfaces.IReactorFDSet interface, but it doesn't declare that it does with zope.interface.

If I've missed something and there's a reason for this omission, a comment would be appreciated...

Attachments:

* [pollreactor-implements-IReactorFDSet.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/464/464f9d1621f0799f4c4b7a2e884b21e4be81d222/0b3033f20a8024a41f285a5e5285c76fe5454549.diff) (688 bytes) - added by mwh on 2006-03-04 13:55:49Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__1539 1539
type__enhancement enhancement
reporter__mwh mwh
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1141480528000000 1141480528000000
changetime__1142096000000000 1142096000000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="foom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|foom set status to `closed`|
|-|-|

(In [16242]) Pollreactor implements IReactorFDSet. Thanks mwh. Fixes [#11073](https://github.com/twisted/twisted/issues/11073).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
